files/es/learn/common_questions/thinking_before_coding/index.html

```diff
@@ -128,38 +128,38 @@ <h3 id="¿Qué_hay_que_hacer_y_en_qué_orden_para_alcanzar_mis_metas">¿Qué hay
     <ol>
      <li>Escribe algunos artículos para iniciar la discusión</li>
      <li>Define cómo deben lucir los artículos</li>
-     <li>Publica los artículos en el sitio web (¿Cómo hacer esto?)</li>
+     <li>Publica esos artículos en el sitio web (¿Cómo hacer esto?)</li>
     </ol>
    </td>
   </tr>
   <tr>
-   <td style="vertical-align: top;">Conocer otros músicos</td>
+   <td style="vertical-align: top;">Conocer a otros músicos</td>
    <td>
     <ol>
-     <li>Provee modos para que las personas te contacten (¿Email? ¿Facebook? ¿Teléfono? ¿Correo?)</li>
+     <li>Provee mecanismos para que las personas te contacten (¿Email? ¿Facebook? ¿Teléfono? ¿Correo?)</li>
```

Comment:
* no es un mecanismo, es una forma de contacto
```suggestion
     <li>Provee formas para que las personas te contacten (¿Email? ¿Facebook? ¿Teléfono? ¿Correo?)</li>
```